jsonschema-core.xml

```diff
@@ -135,13 +135,67 @@
                 data in several ways.
             </t>
             <t>
-                JSON Schema uses keywords to assert constraints on JSON instances or annotate those
-                instances with additional information.  Additional keywords are used to apply
-                assertions and annotations to more complex JSON data structures, or based on
-                some sort of condition.
+                A JSON Schema document is comprised of a set of keywords,
+                which are specified as properties in a JSON object.
+                The name of the keyword is used as the property name,
+                and any arguments to the keyword are provided as the value.
+                Keywords can be classified by their functions.
+                The most important classes are assertion, annotation, and applicator keywords.
```

Comment:
It may be important to recognize that not all keyword provide any of these functions (e.g. `$id`) and some provide more than one (e.g. `properties`).